packages/babel-preset-env/README.md

```diff
@@ -18,22 +18,18 @@ Without any configuration options, @babel/preset-env behaves exactly the same as
 
 You can also configure it to only include the polyfills and transforms needed for the browsers you support. Compiling only what's needed can make your bundles smaller and your life easier.
 
-This example only includes the polyfills and code transforms needed for coverage of users > 0.25%, ignoring Internet Explorer 11 and Opera Mini. We use [browserslist](https://github.com/ai/browserslist) to parse this information, so you can use [any valid query format supported by browserslist](https://github.com/ai/browserslist#queries).
+If you want to specify own target browsers, we recommend to use [`.browserslistrc`](https://github.com/browserslist/browserslist) config. This config is used by many tools including Autoprefixer.
 
-```js
-{
-  "presets": [
-    ["@babel/preset-env", {
-      "targets": {
-        // The % refers to the global coverage of users from browserslist
-        "browsers": [ ">0.25%", "not dead"]
-      }
-    }]
-  ]
-}
+This example only includes the polyfills and code transforms needed for coverage of users > 0.25%, ignoring browsers without security updates like IE Internet Explorer 10 and BlackBerry.
+
+```
+> 0.25%
+not dead
 ```
 
-> You can also target individual versions of browsers instead of using a query with `"targets": { "chrome": "66" }`.
+The full list of queries could be found in [Browserslist docs](https://github.com/browserslist/browserslist#queries).
+
+If you need to use different browsers for Babel, you can specify Browserslist queries in `targets.browsers` option.
```

Comment:
Small edits in bold (also think it would be helpful to link to the target.browsers option):

> If you need to use different browsers for Babel, you can **also** specify Browserslist queries in **the** [`targets.browsers` option](https://github.com/babel/babel/tree/master/packages/babel-preset-env#targetsbrowsers) in your Babel config.